### ISSUE TITLE
Bad output - Grabbing image from 4k.mp4 video

### ISSUE THREAD

**Issue by TristanVALCKE** [jcodec/jcodec#116]
Hello,
thank for this great work !

I got a problem when i extract image from 4k mp4 video, but not whit full-hd mp4.

I'm using this code snippet : 

`
Picture8Bit frame = org.jcodec.api.FrameGrab8Bit.getNativeFrame(inputFile1, frameNumber);
AWTUtil.savePicture(frame, "jpg", new File("frame_12345.jpg"));
`
or

`
BufferedImage frame = FrameGrab8Bit.getFrame(inputFile, i);
ImageIO.write(frame, "jpg", new File("frame_12345.jpg"));
`

The output was the same (for png too)...

![c0016_channel8bit_0](https://cloud.githubusercontent.com/assets/5276909/13106714/0609907a-d56a-11e5-9c31-c8df341febf9.jpg)

Thank in advance,
Tristan.

**Comment by svitvitskiy** [comment]
Thank you,

Could you upload your video so I can have a look?

Stan

**Comment by TristanVALCKE** [comment]
Yes, of course ! I send you (on your jcodec mail) a wetransfere link.
This is juste a test video. If you need other info about this video i remain at disposal.

**Comment by TristanVALCKE** [comment]
Did you receive the file (on jcodecproject@gmail.com) for testing ? Because i got some trouble on upload...

Tristan

**Comment by zhuker** [comment]
@TristanVALCKE sorry the files were deleted can you post a public google drive or dropbox link?

**Comment by Itee** [comment]
I'm sorry i haven't access to this file now ! It was in my last job, sorry... And i can't produce, on my own, 4k video.
